xml/System.Windows.Forms/Application.xml

```diff
@@ -157,7 +157,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- You must attach the event handlers to the <xref:System.Windows.Forms.Application.Exit%2A> event to perform unhandled, required tasks before the application stops running. You can close files opened by this application, or dispose of objects that garbage collection did not reclaim.  
+ You must attach the event handlers to the <xref:System.Windows.Forms.Application.ApplicationExit> event to perform unhandled, required tasks before the application stops running. You can close files opened by this application, or dispose of objects that garbage collection did not reclaim.  
```

Comment:
```suggestion
 You must attach the event handlers to the `ApplicationExit` event to perform unhandled, required tasks before the application stops running. You can close files opened by this application, or dispose of objects that garbage collection did not reclaim.  
```

Reply:
Yes, they are everywhere. The previous build system rendered what would ordinarily be a self-referencing link in bold; the current one renders the link. It's not uncommon for developers reading the page to click on a self-referencing link, which then reloads the page. When we've been encountering self-referencing links, we've tried to fence them.